.github/workflows/setup_microk8s.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This needs to be a separate file as newgrp will not have effect from setup_environment.sh
+microk8s enable storage dns rbac gpu
```

Comment:
less important than the microk8s channel above, but these could be
```suggestion
microk8s enable $MICROK8S_ADDONS
```
so they appear in the workflow like our other workflows